test_files/namespaced/export_enum_in_namespace.ts

```diff
@@ -3,6 +3,11 @@
  * below must be exported onto foo, which tsickle does by disabling its emit for namespace'd enums.
  */
 
+// TODO(#132): namespaces currently don't emit properly.
+// This workaround at least makes them compile.
+// @ts-ignore
+exports = {};
+
 // tslint:disable:no-namespace
 
 export namespace foo {
```

Comment:
I'd prefer to rewrite the code here as

```typescript
namespace foo { ... }
export {foo}
```

as it seems slightly less ugly than the @ts-ignore hack.  The reopened namespace case truly requires that hack, so we'll still have it in our test suite.

Reply:
Done.